partcad/src/partcad/provider_factory_python.py

```
@@ -156,12 +177,10 @@
 
             try:
                 response = base64.b64decode(response_serialized)
-                register_cq_helper()
+                register_ocp_helper()
                 result = pickle.loads(response)
             except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)